db.json

```diff
@@ -0,0 +1,63 @@
+{
+  "events": [
+    {
+      "title": "dasdsa",
+      "weddingDecription": "dsdsa",
```

Comment:
its good, if you added the db in gitignore . Please remove this file from branch as well as the file or data should not be in branch/repo.

Reply:
done file removed